src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+function addMessage(messageText, isError = false) {
+  const message = document.createElement('div');
+
+  message.textContent = messageText;
+  message.classList.add('message', isError ? 'error-message' : null);
```

Comment:
Using `null` in `classList.add` can cause unexpected behavior. Consider using a ternary operator that returns an empty string or a valid class name instead. For example: `isError ? 'error-message' : ''`.